esphome/components/scd30/sensor.py

```diff
@@ -63,6 +71,9 @@
                     min=core.TimePeriod(seconds=1), max=core.TimePeriod(seconds=1800)
                 ),
             ),
+            cv.Optional(CONF_FORCED_CALIBRATION_BASELINE): cv.int_range(
```

Comment:
Please set the default here and remove from the cpp code.
```suggestion
            cv.Optional(CONF_FORCED_CALIBRATION_BASELINE, default=410): cv.int_range(
```